### PR TITLE
Cannot use object of type stdClass as array [mailjet/mailjet-apiv3-ph…

### PR DESCRIPTION
**Pull request by schojniak**
…p/src/Mailjet/Response.php:75] #51

### CODE REVIEW

**Comment by schojniak** [comment]
Can you accept this PR and make a new release?

File: src/Mailjet/Response.php [hunk 1]
```diff
@@ -150,7 +150,7 @@ protected function decodeBody($body)
              */
             $maxIntLength = strlen((string) PHP_INT_MAX) - 1;
             $jsonWithoutBigIntegers = preg_replace('/:\s*(-?\d{'.$maxIntLength.',})/', ': "$1"', $body);
-            $object = json_decode($jsonWithoutBigIntegers);
+            $object = json_decode($jsonWithoutBigIntegers, true);
```
Comment (Benoit-amo) [inline src/Mailjet/Response.php]: I think it should be : 
`$object = json_decode($jsonWithoutBigIntegers, true, 512);`

**Comment by GuillaumeBadi** [comment]
On packagist in a minute! Thanks ! 👍

**Comment by schojniak** [comment]
512 is default value

**Comment by GuillaumeBadi** [comment]
[Yep](http://php.net/manual/fr/function.json-decode.php)

**Comment by schojniak** [comment]
``mixed json_decode ( string $json [, bool $assoc = false [, int $depth = 512 [, int $options = 0 ]]] )``

needed only if you set ``$options``

**Comment by Benoit-amo** [comment]
Ok ok ! :)

**Pull request merged**